lib/shared/messages.js

```diff
@@ -84,16 +84,16 @@ module.exports = {
     },
 
     system () {
-      return 'System Drive'
+      return 'System Target'
     },
 
     containsImage () {
-      return 'Drive Contains Image'
+      return 'Target Contains Image'
     },
 
     // The drive is large and therefore likely not a medium you want to write to.
     largeDrive () {
-      return 'Large Drive'
+      return 'Large Target'
```

Comment:
I'm thinking maybe this could be left as "Large Device", or maybe "Large Storage"?

Reply:
Yeah, "Large Target" makes me think of 
![image](https://user-images.githubusercontent.com/476186/39220024-23ad897a-4826-11e8-9e38-68214f7e0168.png)